frontend/src/component/players-container/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  playersFetchAll,
+  playerCreateRequest,
+  playerUpdateRequest,
+  playerDestroyRequest,
+} from '../../action/player-actions';
+import Player from '../player';
+import PlayerForm from '../player-form';
+
+export class PlayersContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFetching: false,
+    };
+  }
+  componentWillMount() {
+    this.setState({ isFetching: true });
+    this.props.actions.playersFetchAll();
```

Comment:
In addition to the option we discussed in class, you can promisify your playersFetchAll and then add a '.finally()' off playersFetchAll turning the state of isFetching to false. That way no matter if it error'd or was a success it will turn isFetching to false again.